src/hooks/useLogin.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import {useEffect} from "react";
+
+export function useLogin(){
+    useEffect(()=>{
+        (async function loginHandler(){
+            try{
+                const {data} = await axios.post("/api/auth/login",{
+                    email: "johndoe@gmail.com",
+                    password: "johnDoe123"
+                })
+                localStorage.setItem("encodedToken",data.encodedToken);
+                console.log("login success");
```

Comment:
suggestion - you may remove this console.log now.

Reply:
I have added it intentionally to track the flow of the app and how many times the APIs are called. I will remove it after the project is complete.